### PR TITLE
chore(No-JIRA) Adding more 'bricks' to the pile to build the wall

### PR DESCRIPTION
**Pull request by tweigelt-sqsp**
With every brick answered we can build our wall!

### CODE REVIEW

File: src/questionGenerator.js [hunk 1]
```diff
@@ -12,7 +12,37 @@ const questions = [
...
+  'If you were invisible for a day, what would you do?',
+  'What makes you smile when you are having a bad day?',
+  'Would you rather live in the same place for the rest of your life, or have to move to a new country once a month for the next 5 years?',
+  'What is the most spontaneous thing you’ve ever done?',
+  'What was the best time of your life?',
+  'If you got kidnapped, what would you do that would be so annoying that it would make your kidnappers return you?',
+  'Would you rather live in a big city or the middle of nowhere for the rest of your life?',
+  'If you were to spend a day spoiling yourself, what would you do?',
+  'How much of your life would you guess you spend on your phone?',
+  'What keeps you up at night?',
```
Comment (mwall01) [inline src/questionGenerator.js:34]: If anyone walks by the meeting room when this one comes up

File: src/questionGenerator.js [hunk 2]
```diff
@@ -12,7 +12,37 @@ const questions = [
...
+  'If you had to wear one outfit for the rest of your life, what would it be?',
+  'If you had a big decision to make, who would you talk to first?',
```
Comment (mwall01) [inline src/questionGenerator.js:36]: ah jesus

**Review by mwall01 — COMMENTED** [review]
Feels like a therapy session 🤔 😄

**Comment by tweigelt-sqsp** [comment]
Feel free to ask for changes if they do not suit ...

File: src/questionGenerator.js (hunk 2 above)
Comment (dalyw01) [reply src/questionGenerator.js:36]: The answer is @tweigelt-sqsp , setting it up for himself! Well played!

When is the MasterClass?

Comment (tweigelt-sqsp) [reply src/questionGenerator.js:36]: Always good to know whom to ask ;-)

File: src/questionGenerator.js [hunk 3]
```diff
@@ -12,7 +12,37 @@ const questions = [
...
+  'If animals could talk, which one do you think would be the rudest?',
+  'Would you rather be able to speak any language fluently or speak with animals?',
+  'Where do you see yourself in five years from now?',
+  'What is your worst fear?',
+  'If your life was a movie, what would it be called?',
+  'Would you rather be an expert at only 1 thing or average at a lot of things?',
+  'Who was your role model growing up?',
+  'What is the happiest memory that you have?',
+  'What in life are you most passionate about?',
+  'Do you think it’s possible to learn lessons without making mistakes first?',
```
Comment (tkavlearning) [inline src/questionGenerator.js]: Isn't that what PR Reviews are for? 🤣

File: src/questionGenerator.js [hunk 4]
```diff
@@ -12,30 +12,62 @@ const questions = [
...
-//   'What is something interesting about where you grew up?',
-//   'Guilty TV or Movie pleasure, share with the class?',
-//   'What is something interesting about where you grew up?',
-//   'What is your favorite cereal?',
-//   'Which Muppet represents you the best?',
-//   'Would you rather give up Coffee or Pizza?,
-//   'Its the 90s, which Spice Girl are you taking a bullet for?',
-//    'What is your favorite ice cream flavor? Do you prefer a cone or a cup?',
-// ];
+/*
```
Comment (tweigelt-sqsp) [inline src/questionGenerator.js:48]: Just because I like it like that :-D

File: src/questionGenerator.js [hunk 5]
```diff
@@ -12,30 +12,62 @@ const questions = [
...
+  'If you were to spend a day spoiling yourself, what would you do?',
+  'How much of your life would you guess you spend on your phone?',
+  'What keeps you up at night?',
+  'If you had to wear one outfit for the rest of your life, what would it be?',
+  'If you had a big decision to make, who would you talk to first?',
+  'What is one thing you wish you could change about yourself?',
+  'What is a life skill that you always wanted to learn and have never been able to?',
+  'Are there any lessons that you had to learn the hard way?',
+  'What is something that you cannot live without?',
+  'On a scale from one to tenhow would you rate this last week?',
```
Comment (mwall01) [inline src/questionGenerator.js]:
```suggestion
  'On a scale from one to ten how would you rate this last week?',
```

**Review by mwall01 — APPROVED** [review]
Not sure about this but here we go

File: src/questionGenerator.js (hunk 5 above)
Comment (tweigelt-sqsp) [reply src/questionGenerator.js]: Consider it fixed.

Comment (tweigelt-sqsp) [reply src/questionGenerator.js]: Addressed code review change request(s) through:
-  [2258cef](https://github.com/mwall01/weekly-wall/pull/19/commits/2258cef70482c466aa79313ae2ff5a2fecd4ea01)

**Pull request merged**